operations/workplace/people/working-at-mattermost/paid-time-off/README.md

```diff
@@ -2,43 +2,45 @@
-2. In most cases, statutory public holidays for FT Staff in entity locations (Canada, Germany, US, UK) will be reflected in **BambooHR > Time Off** with the exception of specific regional public holidays. If the public holiday is already listed in Bamboo, no action is required. 
-*For all FT Staff, any statutory public holiday in your region of residence that is not listed in BambooHR, please enter the day in **BambooHR > Time Off > Public Holidays** to distinguish it from vacation time.* 
-3. If time off is being taken for sickness, including physical and mental needs, please enter these days in **BambooHR > Sick**, to distinguish from other types of PTO. Follow the appropriate process according to the statutory requirements in your region of residence. If taking extended time off for medical leave or for other types of [leaves](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/leaves-of-absence), reach out to HR: hr@mattermost.com.
-4. PTO requests must be "approved" in BambooHR to show up in the "Who's Out" calendar. Managers are responsible for approving their direct reports' time off requests in BambooHR.  
-5. As long as you have entered your time off in BambooHR, and you have arranged for coverage while you are gone, you are able to take your vacation before it has been approved in BambooHR. More than 14 consecutive days off require manager approval in BambooHR prior to the vacation starting.
-6. Be [vacation-ready](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#vacation-ready): 
+1. Communicate your Paid Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
+2. For audit and compliance purposes, Time Off for EOR staff will also be recorded in Deel by the People team.
+3. Statutory public holidays for FT Staff in entity and EOR locations (Australia, Canada, Germany, Spain, Sweden, US, UK) are automatically listed in Rippling. *For all FT Staff outside of entity and EOR locations, please ensure to enter any statutory public holidays taken in your region of residence in Rippling > Time Off > Public Holiday to distinguish it from vacation time.* 
```

Comment:
please ensure to enter any public holidays in your region of residence in Rippling > Time Off > Public Holiday to distinguish it from Planned Time Off.